grim.go

```diff
@@ -117,10 +117,21 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 		if err != nil {
 			return grimErrorf("error extracting hook from message: %v", err)
 		}
+		logger.Print(hook.Describe() + "\n")
```

Comment:
A better way to do this would be to refactor the if statement to use another function. So like:

```go
if skipReason := shouldSkip(hook); skipReason != nil {
  log.Printf("hook skipped %s: %v\n", skipReason, hook)
return nil
}
```

then the function could return a:
```go
type skipReason interface {
 Stringer
}
```

You can see something similar in the notify.go file with the grimNotification interface.